tutorials/best_practices/scenes_versus_scripts.rst

```diff
@@ -40,7 +40,7 @@ a change in API:
     public partial class Game : Node
     {
         public readonly Script MyNodeScr = (Script)ResourceLoader.Load("MyNode.cs");
-        public readonly PackedScene MySceneScn = (PackedScene)ResourceLoader.Load("MyScene.tscn");
+        public readonly PackedScene MySceneScn = (PackedScene)ResourceLoader.Load("my_scene.tscn");
```

Comment:
I'm not sure if we use snake_case in C# projects. I think it's more common to use PascalCase for all files.

Reply:
I've asked in [Rocket Chat](https://chat.godotengine.org) and I can confirm the snake_case naming convention applies only for GDScript projects, C# projects should likely be using PascalCase for all files.

Reply:
Cool, all done.